api/restHandler/CoreAppRestHandler.go

```diff
@@ -2150,7 +2160,7 @@ func (handler CoreAppRestHandlerImpl) GetAppWorkflow(w http.ResponseWriter, r *h
 	}
 
 	//get/build app workflows starts
-	appWorkflows, err, statusCode := handler.buildAppWorkflows(appId)
+	appWorkflows, err, statusCode := handler.buildAppWorkflows(appId, "")
```

Comment:
add comments why empty string. and use variable for empty string.

Reply:
done